exporters/module_loader.py

```diff
@@ -48,6 +48,14 @@ def load_stats_manager(self, options, metadata, **kwargs):
         from exporters.stats_managers.base_stats_manager import BaseStatsManager
         return self._load_module(options, metadata, BaseStatsManager, **kwargs)
 
+    def load_write_buffer(self, module_name, **kwargs):
+        from exporters.write_buffer import WriteBuffer
+        cls = self.load_class(module_name)
+        instance = cls(**kwargs)
+        if not isinstance(instance, WriteBuffer):
+            raise TypeError('Module must inherit from ' + str(WriteBuffer))
```

Comment:
also, per the other comment, we need there to load the class before instantiating.